alignak/action.py

```diff
@@ -136,7 +136,6 @@ class ActionBase(AlignakObject):
     def __init__(self, params=None, parsing=True):
         super(ActionBase, self).__init__(params, parsing=parsing)
         self.creation_time = time.time()
```

Comment:
Yes, it may be better! I saw this this morning when I ran the test once more ...